js/libs/builder.js

```diff
@@ -191,11 +196,117 @@ var app = new Vue({
     },
     save: function() {
       return Fliplet.Widget.save(this.settings);
+    },
+    configureEmailTemplate: function() {
+      var $vm = this;
+      var allFields = $vm.settings.fields;
+      // Creates default email template
+      var defaultEmailTemplate = '<h1>' + $vm.settings.displayName + '</h1><p>A new form submission has been received.</p>';
+      defaultEmailTemplate += '<ul>';
+      for (var i = 0; i < allFields.length; i++) {
```

Comment:
can't you just use `$vm.settings.fields.forEach` and avoid using a `for` cycle and `allFields[i]` multiple times?